app/assets/javascripts/proxy_status.js

```diff
@@ -1,3 +1,8 @@
+$(document).on('ContentLoad', function() {
+  showProxies();
```

Comment:
No, the spinner is loaded already in the html, but since only the html is replaced when submitting with two-pane, the `document.load` event is not fired.